rllib/evaluation/postprocessing.py

```diff
@@ -188,7 +188,7 @@ def compute_gae_for_sample_batch(
             policy.model.view_requirements, index="last"
         )
 
-        if policy.config["_enable_rl_module_api"]:
+        if policy.config.get("_enable_rl_module_api"):
```

Comment:
This creates backward compatibility issues, since older configs don't have this attribute.
Since these configs now are not merged with today's defaults, we'd run into an issue here.